app/models/manageiq/providers/cisco_intersight/manager_mixin.rb

```diff
@@ -1,9 +1,8 @@
-require 'intersight_client'
-
 module ManageIQ::Providers::CiscoIntersight::ManagerMixin
   extend ActiveSupport::Concern
 
   def connect(options = {})
+    require 'intersight_client'
```

Comment:
I could be wrong, but I don't think this is needed because raw_connect already doesn't the require

Reply:
Yeah, you're right.  I did the require here because we use the internals of the client on line 15 and rely on the `raw_connect` side effect of loading it. I can drop it but thought it wasn't obvious that it would be required for you.

Reply:
Ahh ok - I didn't see the safe_constantize...that's fine then